README.md

```diff
@@ -1 +1,103 @@
-# customerio-reactnative
\ No newline at end of file
+<p align="center">
+  <a href="https://customer.io">
+    <img src="https://user-images.githubusercontent.com/6409227/144680509-907ee093-d7ad-4a9c-b0a5-f640eeb060cd.png" height="60">
+  </a>
+  <p align="center">Power automated communication that people like to receive.</p>
+</p>
+
+![min swift version is 5.3](https://img.shields.io/badge/min%20Swift%20version-5.3-orange)
+![min ios version is 13](https://img.shields.io/badge/min%20iOS%20version-13-blue)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](code_of_conduct.md) 
+[![codecov](https://codecov.io/gh/customerio/customerio-reactnative/branch/main/graph/badge.svg?token=IZ9RP9XD1O)](https://codecov.io/gh/customerio/customerio-reactnative)
+
+# Customer.io React Native SDK
+
+This is the official Customer.io SDK for React Native.
+
+You'll find our [complete SDK documentation at https://customer.io/docs/sdk/react-native](https://customer.io/docs/sdk/react-native/). This readme only contains basic information to help you install and initialize the SDK.
+
+## Install and initialize the SDK
+
+1. Open your terminal and run `npm install customerio-reactnative`
+1. Add iOS dependencies to your project by going to the iOS subfolder and running `pod install`. 
```

Comment:
Minor fix - Counter is always 1 for all points/steps.

Reply:
I mean, it seems likely but I'm not sure it's a problem. Should be some "standard" we can find to make sure this is consistent with the rest of the world.

Reply:
Hmmm....
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists

Reply:
i've seen this before in other repos too.
because some of the items are multi-line i think we should stick with `1.` since it makes it easier for us.

dont think we need an explanation but if wanted could add a comment stating that when rendered numbers auto-increment https://www.markdownguide.org/basic-syntax/#ordered-lists